src/module/vue/components/sf-moverow.vue

```diff
@@ -62,9 +62,7 @@
 @import (reference) '../../../styles/mixins.less';
 
 @icon_size: 1.2em;
-@border_width: var(--ironsworn-border-width-lg);
-@border_radius: var(--ironsworn-border-radius-lg);
-@wrapper_spacing: 4px;
+@wrapper_spacing: var(--ironsworn-spacer-md);
```

Comment:
Looks like this one can go away? All references to this variable seem to have been replaced with `var(--ironsworn-spacer-md)`.